django/db/models/query.py

```diff
@@ -319,15 +319,10 @@ def aggregate(self, *args, **kwargs):
         if self.query.distinct_fields:
             raise NotImplementedError("aggregate() + distinct(fields) not implemented.")
         for arg in args:
-            # The default_alias property may raise a TypeError, so we use
```

Comment:
If not switching to `hasattr()`, keeping a shortened version of the comment could be appropriate since the code isn't extremely obvious about where and why the exception is raised.